core/src/main/java/org/htsjdk/core/utils/PathSpecifier.java

```diff
@@ -0,0 +1,238 @@
+package org.htsjdk.core.utils;
```

Comment:
Either `org.htsjdk.core.utils.io` or in the same package as the `PathURI` (or the renamed version).

Reply:
Done.